FlashCap.Core/Internal/NativeMethods_DirectShow.cs

```diff
@@ -263,7 +263,7 @@ public interface IEnumFilters
     {
         [PreserveSig] int Next(
             int request,
-            [Out, MarshalAs(UnmanagedType.LPArray, SizeParamIndex=0)] IPin?[] filters,
+            [Out, MarshalAs(UnmanagedType.LPArray, SizeParamIndex=0)] IBaseFilter?[] filters,
```

Comment:
I found out this typing error while experimenting

Reply:
@ldlac Thanks, it is my mistake and your changes are correct.

https://learn.microsoft.com/en-us/windows/win32/api/strmif/nf-strmif-ienumfilters-next